proto/aquila/action_communication.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+import "runtime_function_definition.proto";
+
+option ruby_package = "Tucana::Aquila";
+
+package aquila;
+
+// Event that gets admitted by an action
+message Event {
+  // Id of Event type
+  string event_type = 1;
+  // Payload (JSON) of event params
+  string payload = 2;
+}
+
+// Action flow/event configuration
+message Configuration {
+  // Action identifier
+  string identifier = 1;
+  // Configuration
+  repeated shared.RuntimeFunctionDefinition function_definition = 2;
+}
+
+// Request to execute a request a flow
+message ExecutionRequest {
+  // Execution identifier of execution
+  string execution_identifier = 1;
+  // Function identifier of flow to execute
+  string function_identifier = 2;
+  // Parameters of flow required to execute
+  repeated string parameters = 3;
+}
+
+// Result from executed flows by an action
+message ExecutionResult {
+  // Identifier of flow to execute
+  string execution_identifier = 1;
+  // Result of executed flow
+  repeated string result = 2;
```

Comment:
Instead of a string, this could be a [`google.protobuf.Value`](https://protobuf.dev/reference/protobuf/google.protobuf/#value) to remove some need of parsing